gpt4all-chat/metadata/models3.json

```diff
@@ -13,7 +13,8 @@
     "description": "<ul><li>Fast responses</li><li>Chat based model</li><li>Large context size of 128k</li><li>Accepts agentic system prompts in Llama 3.1 format</li><li>Trained by Meta</li><li>License: <a href=\"https://llama.meta.com/llama3_1/license/\">Meta Llama 3.1 Community License</a></li></ul>",
     "url": "https://huggingface.co/GPT4All-Community/Meta-Llama-3.1-8B-Instruct-128k/resolve/main/Meta-Llama-3.1-8B-Instruct-128k-Q4_0.gguf",
     "promptTemplate": "<|start_header_id|>user<|end_header_id|>\n\n%1<|eot_id|><|start_header_id|>assistant<|end_header_id|>\n\n%2",
```

Comment:
While we're here... why does the prompt template specify that user messages end with `<|eot_id|>` but (previous) assistant messages don't? The Llama 3 template appends it to both.

Reply:
Not entirely sure, but could it be because 3.1 introduces <|eom_id|> ?